packages/zowe-explorer/src/dataset/actions.ts

```diff
@@ -66,7 +66,7 @@ export async function allocateLike(datasetProvider: Types.IZoweDatasetTreeType,
         quickpick.ignoreFocusOut = true;
 
         for (const thisSession of datasetProvider.mSessionNodes) {
-            if (!thisSession.label.toString().includes("Favorites")) {
+            if (!thisSession.label.toString().includes(vscode.l10n.t("Favorites"))) {
```

Comment:
Thanks for updating these 😋 
I'm sure we may have a few more out there, but we can address them as we find them 😋